airbyte-integrations/connectors/source-box-data-extract/README.md

```diff
@@ -0,0 +1,105 @@
+# Box Generated Source
+
+This is the repository for the Box Generated source connector, written in Python.
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.com/integrations/sources/box-generated).
+
+## Local development
+
+### Prerequisites
+
+* Python (`^3.9`)
```

Comment:
I know this is straight from the template, but these are outdated. It should be Python 3.10 and Poetry 1.8

Reply:
Bumped to python 3.11 and poetry 1.8